src/store/modules/game.js

```diff
@@ -23,6 +23,7 @@ function resetState() {
     waitingForOpponentToCounter: false,
     myTurnToCounter: false,
     isRanked: false,
+    isSpectating: false,
```

Comment:
I'm leaning towards taking this out of the store and making it a computed prop in gameView based on the route. I don't think it's used outside the GameView component itself so that seems reasonable.

Reply:
Moved into computed prop; I figure if it _can_ be computed, it's generally better that it be computed to prevent potential desync